lighthouse-core/audits/user-timings.js

```diff
@@ -118,6 +118,7 @@ class UserTimings extends Audit {
         const time = item.isMark ? item.startTime : item.duration;
         return {
           name: item.name,
+          startTime: item.startTime,
```

Comment:
can you assert this in user-timing-test?

Reply:
done